docs/usage/shooted_seed.md

```diff
@@ -1,6 +1,10 @@
 # Create Shooted Seed Cluster
 
-Create shooted seed cluster with the `shoot.gardener.cloud/use-as-seed` annotation.
+Create managed seed (aka "shooted seed") cluster with the `shoot.gardener.cloud/use-as-seed` annotation.
+
+**Note:** Starting with Gardener v1.18, the `shoot.gardener.cloud/use-as-seed` annotation is deprecated. 
```

Comment:
Can you also mention this in the release note?

Reply:
Done